.eslintrc.js

```diff
@@ -26,7 +25,10 @@ module.exports = {
       }
     ],
     '@typescript-eslint/camelcase': 'off',
-    '@typescript-eslint/explicit-function-return-type': 'off'
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    'react/prop-types': 'off',
```

Comment:
Unfortunately, we haven't migrated away from using `prop-types` yet, even though we currently use typescript. This is something we would likely consider doing in the future.

Reply:
fixed